minutes/2023-05-23.md

```diff
@@ -0,0 +1,27 @@
+# Parsec Community Meeting (23/05/2023) 📋
+
+Minutes of previous meeting are stored
+[here](https://github.com/parallaxsecond/community/tree/main/minutes).
+
+## Attendance
+
+- Adam Johnston (@adamjohnstonarm)
+- Ionut Mihalcea (@ionut-arm)
+- Paul Howard (@paulhowardarm)
+- Gowtham Suresh Kumar (@gowthamsk-arm)
+- Vikas Katariya
+
+## Agenda
+
+- RSA key generation issue with trusted services backend
+
+## Minutes
+
+- When running parsec tool cli test script against the parsec service configured with trusted
+   services backend on N1SDP platform, the script terminates with a would block OS error. Gowtham
```

Comment:
"would block OS" error may be clearer?

Reply:
> # [WARN ] Conversion from Os { code: 11, kind: WouldBlock, message: \"Resource temporarily unavailable\" } to ResponseStatus::ConnectionError.# [WARN ] The WouldBlock error might mean that the connection timed out. 
This is the error log :)

Reply:
I just meant to use the quotes.
"Resource temporarily unavailable, WouldBlock" OS error if we want to be more specific :)